src/slacko.ml

```diff
@@ -1075,8 +1133,14 @@ let chat_delete token ts chat =
     | #message_error as res -> res
     | _ -> `Unknown_error
 
+let jsonify_attachments = function
```

Comment:
Yes. This is one of the places where I didn't put much thought into the code beyond "does this work at all", so it could definitely use some more attention in general.